src/mono/wasm/debugger/BrowserDebugProxy/JObjectValueCreator.cs

```diff
@@ -265,22 +265,51 @@ async Task<string> GetNullObjectClassName()
             return className;
         }
     }
-
+    private async Task<string> ToString(List<int> typeIds, int objectId, CancellationToken token)
+    {
+        string description = "";
+        try {
+            foreach (var typeId in typeIds)
+            {
```

Comment:
Can we extract the body of the loop to `MonoSDBHelper`, and use that here, and in `ValueTypeClass`?

Reply:
done